types/node/fs.d.ts

```diff
@@ -2544,6 +2544,12 @@ declare module "fs" {
         options: ReadAsyncOptions<TBuffer>,
         callback: (err: NodeJS.ErrnoException | null, bytesRead: number, buffer: TBuffer) => void,
     ): void;
+    export function read<TBuffer extends NodeJS.ArrayBufferView>(
+        fd: number,
+        buffer: TBuffer,
+        options: ReadSyncOptions,
+        callback: (err: NodeJS.ErrnoException | null, bytesRead: number, buffer: TBuffer) => void,
+    ): void;
```

Comment:
It [looks like](https://nodejs.org/docs/latest/api/fs.html#fsreadfd-buffer-options-callback) this also needs an additional signature without the optional `options` parameter.

Reply:
Thanks! I added in [314e72d](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/72217/commits/314e72d4643980ab912c89d85329cc54a4190bea).